.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build and deploy WCAG Primer
+
+on:
+- push
+
+jobs:
+  # A really basic check to see if this builds okay.
+  only-build:
+    runs-on: ubuntu-latest
+    if: github.ref != 'refs/heads/main' || github.ref != 'refs/heads/master'
```

Comment:
As these two jobs are not dependent on each other, would it be worth splitting this into two jobs and using [`on.push.branches`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags)? I think that might be slightly clearer.

Reply:
Now might be as good a time as any to rename the default branch here, too.

Reply:
I like the splitting into two jobs a lot better - feels less flakey than the current `if` statement. 

I'm going to leave the renaming of the branch to another pull request to avoid any unnecessary complications.

Reply:
This has been split into two jobs - `build` for testing whether this builds and `deploy` for, uh, deploying it :)